scripts/aggregate-raw-data.R

```diff
@@ -248,6 +248,10 @@ all_data$GlyphID <- get_codepoints(all_data$Phoneme)
 ## CONVERT INVENTORY ID TO INTEGER
 all_data$InventoryID <- as.integer(all_data$InventoryID)
 
+## SORT BEFORE SAVING TO ELIMINATE SPURIOUS GIT DIFFS
+sort_order <- with(all_data, order(InventoryID, SegmentClass, GlyphID))
```

Comment:
@drammock - so this sorts by InventoryID, then consonants < tones < vowels, then by GlyphID. but the GlyphIDs are treated as strings and not decimal ordering, I think:

>>> int("006A", 16)
106
>>> int("0068", 16)
104
>>> int("0077", 16)
119
>>> int("0061", 16)
97

If it's consistent throughout, I'm fine with it as is.

Reply:
GlyphID has to be treated as a string, because of cases like `006B+02B0`. If you want to sort by something else instead (phoneme?) I can change it.  As far as storing in string vs decimal integer or hexadecimal integer, I think string is best because it straightforwardly reflects unicode codepoint values that people are likely to encounter elsewhere.  Converting the representation of `kʰ` from `006B+02B0` to something like `107,688` seems like a move away from usability in my opinion.